client/lib/i18n-utils/utils.js

```diff
@@ -54,6 +54,26 @@ const i18nUtils = {
 	},
 
 	/**
+	 * Given the content of an 'AcceptedLanguages' request header, returns an array of the languages.
```

Comment:
This looks like a server function (working on request headers), it should live in `/server` I think